### PR TITLE
SONARJS-586 Update message of DuplicateFunctionArgument

### PR DESCRIPTION
**Pull request by vilchik-elena**

### CODE REVIEW

File: javascript-checks/src/main/java/org/sonar/javascript/checks/DuplicateFunctionArgumentCheck.java [hunk 1]
```diff
@@ -43,28 +45,47 @@
...
+  }
 
-    if (values.contains(unescaped)) {
-      getContext().addIssue(this, identifier, String.format(MESSAGE, value));
-    } else {
-      values.add(unescaped);
+  private static String parameterListString(List<String> duplicatedParameters) {
+    String duplicatedParametersString = "";
+    for (String duplicatedParameter : duplicatedParameters) {
+      duplicatedParametersString += String.format("\"%s\", ", duplicatedParameter);
```
Comment (SonarTech) [inline javascript-checks/src/main/java/org/sonar/javascript/checks/DuplicateFunctionArgumentCheck.java]: MAJOR Use a StringBuilder instead. [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1643)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

File: javascript-checks/src/main/java/org/sonar/javascript/checks/DuplicateFunctionArgumentCheck.java [hunk 2]
```diff
@@ -43,28 +45,47 @@
...
+      getContext().addIssue(this, tree, message);
+    }
+  }
 
-    if (values.contains(unescaped)) {
-      getContext().addIssue(this, identifier, String.format(MESSAGE, value));
-    } else {
-      values.add(unescaped);
+  private static String parameterListString(List<String> duplicatedParameters) {
+    StringBuilder duplicatedParametersString = new StringBuilder();
```
Comment (pynicolas) [inline javascript-checks/src/main/java/org/sonar/javascript/checks/DuplicateFunctionArgumentCheck.java]: @vilchik-elena Why not using `com.google.common.base.Joiner`? e.g. `return "\"" + Joiner.on("\", \"").join(duplicatedParameters) + "\"";`

Comment (vilchik-elena) [inline javascript-checks/src/main/java/org/sonar/javascript/checks/DuplicateFunctionArgumentCheck.java]: @pynicolas Thanks! fixed in 4fd337b

**Pull request merged**